app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt

```diff
@@ -201,7 +201,10 @@ class AudioService : LifecycleService() {
                             .build()
                 }
             })
-
+            // override media connector increment speed
```

Comment:
Please add a new line before the comment

Reply:
Added New line